addons/website_sale/views/snippets/snippets.xml

```diff
@@ -96,9 +101,9 @@
         <we-button data-set-search-type="products" data-select-data-attribute="products" data-name="search_products_opt" data-form-action="/shop">Products</we-button>
     </xpath>
     <xpath expr="//div[@data-js='SearchBar']/we-select[@data-name='order_opt']" position="inside">
-        <we-button data-set-order-by="list_price asc" data-select-data-attribute="list_price asc" data-dependencies="search_products_opt" data-name="order_price_asc_opt">Price (low to high)</we-button>
-        <we-button data-set-order-by="list_price desc" data-select-data-attribute="list_price desc" data-dependencies="search_products_opt" data-name="order_price_desc_opt">Price (high to low)</we-button>
-        <we-button data-set-order-by="website_sequence asc" data-select-data-attribute="website_sequence asc" data-dependencies="search_products_opt" data-name="order_sequence_asc_opt">Sequence</we-button>
+        <t t-foreach="request.env['website']._get_product_sort_mapping()" t-as="query_and_label">
+            <we-button t-if="'name' not in query_and_label[0]" t-att-data-set-order-by="query_and_label[0]" t-att-data-select-data-attribute="query_and_label[0]" data-dependencies="search_products_opt"><t t-out="query_and_label[1]"/></we-button>
```

Comment:
t-if="'name' not int ... ? 

What is the purpose...
Seems verrry louch and not multi lang ready

Reply:
Yes, I know it's a bit ugly 🤣 Basically the sorting list for products I have is inserted into an existing list for "general sorting" which already has "name" sorting (name ASC and name DESC). So if the check isn't there, there are duplicate choices in the dropdown for name sorting (twice name A-Z and name Z-A). I decided to exclude the name sorting I have in the list of sorting for products, because there is no point (and it's ugly) to have it twice in the dropdown. There might be a cleaner way to do it, do you have something in mind?

Reply:
big ugly
btw, we can remove `name  Z -> A` ?

Reply:
That's a PO decision I think, there are certain shops where people might want to order items in reverse naming order IMO, even thought I've never done it myself 🤔

Reply:
Also it exists for the general sorting lists, and removing it from the product sorting list of this PR wouldn't remove it from the other places where it exists.

Reply:
ok can we be more explicit on this hack in this case...

```xml
<!-- name asc and desc are already in default sort by of this snippet. -->
<we-button t-if="query_and_label[0] not in ('name asc', 'name desc')" ...
```